.mergify.yml

```diff
@@ -0,0 +1,69 @@
+pull_request_rules:
+  - name: remove outdated reviews for non trusted authors
+    conditions:
+      - base=master
+      - author!=@mozilla/application-services
+    actions:
+      dismiss_reviews:
+        message: The pull request has been modified, dismissing previous reviews.
+      label:
+        remove:
+          - checkin-needed
+  - name: automatic merge for master
+    conditions:
+      - "#approved-reviews-by>=1"
+      - base=master
+      - label=checkin-needed
+      - "status-success=android-build-pr"
+      - "status-success=lint-detekt"
+      - "status-success=lint-ktlint"
+      - "status-success=ci/circleci: Check Protobuf files are up-to-date"
+      - "status-success=ci/circleci: Check Rust dependencies"
+      - "status-success=ci/circleci: Check Rust formatting"
+      - "status-success=ci/circleci: Check Swift formatting"
+      - "status-success=ci/circleci: Lint Bash scripts"
+      - "status-success=ci/circleci: Lint Rust with clippy"
+      - "status-success=ci/circleci: Rust benchmarks"
+      - "status-success=ci/circleci: Rust tests"
+      - "status-success=ci/circleci: Sync integration tests"
+      - "status-success=ci/circleci: iOS build and test"
```

Comment:
Github appears to have eaten my comment, re-posting:

If we add another suite of checks to CI, do we need to remember to come here and add it to this file as well? Seems OK to start but I'd worry about it getting out of sync over the long haul.

Reply:
Yes, sadly ["There is no such thing as “every status check” in GitHub."](https://doc.mergify.io/conditions.html#validating-all-status-check). I'll add some warnings to our CI files.